### ISSUE TITLE
TreeListEditor exception

### ISSUE THREAD

**Issue by genetsEvgen** [eXpandFramework/eXpand#457]
TreeListEditor
AllowEdit - true

   в System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   в System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   в System.String.System.IConvertible.ToInt32(IFormatProvider provider)
   в System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   в System.Convert.ChangeType(Object value, Type conversionType)
   в Xpand.ExpressApp.TreeListEditors.Win.Controllers.TreeListInplaceEditViewController.treeList_CellValueChanged(Object sender, CellValueChangedEventArgs e)
   в DevExpress.XtraTreeList.TreeList.RaiseCellValueChanged(CellValueChangedEventArgs e)
   в DevExpress.XtraTreeList.TreeList.OnSetValue(TreeListNode node, Object columnID, Object val)
   в DevExpress.XtraTreeList.Nodes.TreeListNode.SetValue(Object columnID, Object val)
   в DevExpress.ExpressApp.Win.Controls.ObjectTreeList.SetNodeValue(ObjectTreeListNode controlNode, TreeListColumn column)
   в DevExpress.ExpressApp.Win.Controls.ObjectTreeList.OnControlNodeBuild(ObjectTreeListNode controlNode)
   в DevExpress.ExpressApp.Win.Controls.ObjectTreeList.BuildControlNode(Object nodeObject, ObjectTreeListNode parentNode)
   в DevExpress.ExpressApp.Win.Controls.ObjectTreeList.LoadFromDataSource()
   в DevExpress.ExpressApp.Win.Controls.ObjectTreeList.UpdateDataSource(Boolean updateContent)
   в DevExpress.XtraTreeList.TreeList.UpdateDataSourceOnChangeDataSource()
   в DevExpress.XtraTreeList.TreeList.set_DataSource(Object value)
   в DevExpress.ExpressApp.Win.Controls.ObjectTreeList.set_DataSource(Object value)
   в DevExpress.ExpressApp.TreeListEditors.Win.TreeListEditor.UpdateControlDataSource()
   в DevExpress.ExpressApp.TreeListEditors.Win.TreeListEditor.treeList_HandleCreated(Object sender, EventArgs e)
   в System.EventHandler.Invoke(Object sender, EventArgs e)
   в System.Windows.Forms.Control.OnHandleCreated(EventArgs e)
   в DevExpress.XtraEditors.Container.EditorContainer.OnHandleCreated(EventArgs e)
   в DevExpress.XtraTreeList.TreeList.OnHandleCreated(EventArgs e)
   в System.Windows.Forms.Control.WmCreate(Message& m)
   в System.Windows.Forms.Control.WndProc(Message& m)
   в DevExpress.XtraEditors.Container.EditorContainer.WndProc(Message& m)
   в DevExpress.XtraTreeList.TreeList.WndProc(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   в System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by apobekiaris** [comment]
please post a sample that I can repro the behavior

**Comment by genetsEvgen** [comment]
Any TreeListEditor with AllowEdit = true

**Comment by genetsEvgen** [comment]
Hello. Do you have a solution?

**Comment by apobekiaris** [comment]
You can disable the bad controller until it is fixed by adding the next line in your Program.cs. 
```cs
XpandModuleBase.DisableControllers(typeof(TreeListInplaceEditViewController));
```

**Comment by apobekiaris** [comment]
I hope u already disabled the controller or have a workaround. The issue however looks like a XAF bug and should be fixed in 19.1.5
https://www.devexpress.com/Support/Center/Question/Details/T757657/issue-with-e443-treelisteditor-how-to-enable-in-place-editing-since-upgrading-to-19-1

**Comment by expand** [comment]
Closing issue for age. Feel free to reopen it at any time.

.Thank you for your contribution.
